src/conformance/framework/conformance_utils.cpp

```diff
@@ -1025,7 +1025,9 @@ namespace Conformance
             case XR_SESSION_STATE_FOCUSED: {
                 // In these states we need to submit frames. Otherwise the runtime won't
                 // necessarily move us from synchronized to visible or focused.
-                REQUIRE(SubmitFrame() == RunResult::Success);
+                if ((autoBasicSession->optionFlags & AutoBasicSession::skipGraphics) == 0) {
```

Comment:
Might consider using a getter instead of direct data access, something like "IsSkippingGraphics" ?

Reply:
I had some thoughts about a more general `IsOptionFlagSet` but I don't think we need to check any other flags anywhere so a `bool SkipGraphics()` would be a bit better to keep the `optionFlags` read only.